### PR TITLE
Api calls

### PR DESCRIPTION
**Pull request by kotesh-arya**
- Multiple  API calls for mock backed for state management of the App

### CODE REVIEW

**Review by Tejs1 — APPROVED** [review]
You can Add Categories in Constant files

File: src/contexts/playlistVideosContext.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+//         type: "PLAYLIST_VIDEOS",
+//         // payload: response.data.likes,
+//       });
+//       console.log(response, "response from delete request");
+//     } catch (e) {
+//       console.log("error during remove from likes:", e);
+//     }
+//   };
+
+//   return (
```
Comment (swethaogeti) [inline src/contexts/playlistVideosContext.js:82]: remove commented code !

**Review by swethaogeti — APPROVED** [review]
remove commented code . good to merge

File: src/contexts/authContext.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    userNotFound: false,
+    userExists: false,
+  };
+  const [userState, userDispatch] = useReducer(reducer, intialState);
+  // const { dispatch } = GlobalCartContext();// this dispatching is for the modal operating used in cart reducer of E-commerce :)
+
+  // Login Function
+
+  const loginUser = async (email, password) => {
+    // console.log(email, password);
```
Comment (Ramandeep9898) [inline src/contexts/authContext.js:25]: remove all console.logs

File: src/contexts/historyContext.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import { createContext, useContext, useReducer, useEffect } from "react";
+import { historyReducer } from "../reducers/historyReducer";
+import axios from "axios";
+import { useAuth } from "./authContext";
+const HistoryContext = createContext();
+
+const HistoryProvider = ({ children }) => {
+  const { token } = useAuth();
+  // console.log(token);
```
Comment (Ramandeep9898) [inline src/contexts/historyContext.js:9]: here too

**Review by Ramandeep9898 — COMMENTED** [review]
Code Looks Clean.

File: src/contexts/authContext.js (hunk 2 above)
Comment (kotesh-arya) [reply src/contexts/authContext.js:25]: sure

File: src/contexts/playlistVideosContext.js (hunk 1 above)
Comment (kotesh-arya) [reply src/contexts/playlistVideosContext.js:82]: Sure 👍🏻

**Pull request merged**